pkg/security/certificate_loader_test.go

```diff
@@ -167,6 +167,13 @@ func makeTestCert(
 func TestNamingScheme(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 
+	permissionRequirement := ".* exceeds -rwx------"
+
+	processID := os.Getuid()
+	if processID == 0 {
+		permissionRequirement = "exceeds -rwxr-----"
```

Comment:
A comment here would be great -- it is not clear at first glance why this would be necessary if we're the root user.

Reply:
Done! I want to add some description about this to the CockroachDB docs but I haven't looked into how to do that yet 😅